packages/commercetools/composables/src/helpers/internals/enhanceCategory.ts

```diff
@@ -0,0 +1,30 @@
+import { ApolloQueryResult } from 'apollo-client'
```

Comment:
maybe mixins is a better name instead of "internals"?

Reply:
I was thinking about the name for these "private" fields, and I think `internals` is the best one as it describes the purpose - they are fields or logic that we will use **internally**  it could be the mixins but it might be confusing with the vue mixins. I'll keep it as it is now (intenals) but maybe we will come up with a different name